.travis.yml

```diff
@@ -18,6 +18,11 @@ matrix:
     env: JRUBY_OPTS='--dev'
   - rvm: *oldest_ruby
     env: ROUGE_VERSION='~> 2.0.0'
+  - rvm: truffleruby-19.2.0
+    if: env(TRAVIS_EVENT_TYPE) = cron
```

Comment:
Go ahead and comment out this line for the purpose of this PR.

Reply:
Also, I believe the line should be:

```
if: type = cron
```

Reply:
The documentation is really confusing:

> To check whether a build was triggered by cron, examine the TRAVIS_EVENT_TYPE environment variable to see if it has the value cron.

https://docs.travis-ci.com/user/cron-jobs/#detecting-builds-triggered-by-cron

> The following known attributes are available, and can be used in conditions:
type (the current event type, known event types are: push, pull_request, api, cron)

https://docs.travis-ci.com/user/conditions-v1#integration

So according to the documentation both should work but `if: type = cron` is more concise (and easier to read).

Reply:
I also tested the TRAVIS_EVENT_TYPE and found it didn't work. But `type = cron` did work.